roles/sap_ha_install_anydb_ibmdb2/tasks/platform/db2cm_cloud_msazure_vm.yml

```diff
@@ -58,15 +76,19 @@
   failed_when: not __sap_ha_install_anydb_ibmdb2_config_msazure_1.rc == 0 and not 'ip address is pingable' in __sap_ha_install_anydb_ibmdb2_config_msazure_1.stdout
 
 # Setting to be performed on both Primary and Secondary node
-- name: SAP HA AnyDB - IBM Db2 HADR (MS Azure) - Primary and Secondary Node - Provide MS Azure CLI credentials
+# Reasons for noqa:
+# - no-changed-when: Db2 mandates use of AWS CLI for credentials instead of credential configuration directly via IBM Db2 selected Pacemaker Shell (crmsh)
+- name: SAP HA AnyDB - IBM Db2 HADR (MS Azure) - Primary and Secondary Node - Provide MS Azure CLI credentials # noqa no-changed-when
   register: __sap_ha_install_anydb_ibmdb2_config_msazure_credentials
   when:
     - __sap_ha_install_anydb_ibmdb2_platform == "cloud_msazure_vm"
   ansible.builtin.shell: |
```

Comment:
Shouldn't the Ansible module be in the line after the line starting with `name:`? Or is there a specific reason for this order of the `shell` module and the `register:` and `when:` lines?